telegram/start_telegram_bot.js

```diff
@@ -272,13 +272,19 @@ module.exports = (args, cbk) => {
           {command: 'mempool', description: 'Get info about the mempool'},
           {command: 'pay', description: 'Pay a payment request'},
           {command: 'pending', description: 'Get pending forwards, channels'},
+          {command: 'stop', description: 'Stop the bot'},
           {command: 'version', description: 'View current bot version'},
         ]);
 
         bot.catch(err => logger.error({telegram_error: err}));
 
         bot.use(async (ctx, next) => {
           try {
+          // Ignore messages that are old
```

Comment:
is this still needed since the bot shuts down gracefully?

Reply:
If the process crashes and auto restarts, it responds to older messages on restart, so left it. 
Can probably be removed because it doesn't work if the auto-restart is under 10 seconds.